portal-impl/src/com/liferay/portal/upgrade/UpgradeProcess_7_0_3.java

```diff
@@ -36,6 +37,7 @@ public int getThreshold() {
 	protected void doUpgrade() throws Exception {
 		upgrade(UpgradeMessageBoards.class);
 		upgrade(UpgradeModules.class);
+		upgrade(UpgradeOrganization.class);
```

Comment:
Hey @JorgeFerrer, I've just realised this line is not properly ordered. It should be after UpgradeOracle

Sorry!